src/main/java/com/google/ehub/data/CommentDataManager.java

```diff
@@ -19,19 +19,25 @@
   public static final String ITEM_ID_PROPERTY_KEY = "itemId";
   public static final String COMMENT_PROPERTY_KEY = "comment";
   public static final String TIMESTAMP_PROPERTY_KEY = "timestampMillis";
+    public static final String EMAIL_PROPERTY_KEY = "email";
```

Comment:
Nit: Fix spacing at the left of ``public static`` also do these constants have to be public?

Reply:
Some of the fields are public because they are shared with other classes. But maybe not all of them need to be